### PR TITLE
Feature/multi line input

### PR DESCRIPTION
**Pull request by ElishaSamPeterPrabhu**
Screenshot_1687419671
Screenshot_1687419692

### CODE REVIEW

**Review by BAL4-R — DISMISSED** [review]

File: DemoApp/DemoApp/DemoApp.csproj [hunk 1]
```diff
@@ -111,4 +117,10 @@
 	  </MauiXaml>
 	</ItemGroup>
 
+	<ItemGroup>
+	  <None Remove="Trial\" />
```
Comment (rthanga1) [inline DemoApp/DemoApp/DemoApp.csproj:121]: Do we need this?

File: DemoApp/DemoApp/MainPage.xaml [hunk 2]
```diff
@@ -1,9 +1,15 @@
 ﻿<?xml version="1.0" encoding="utf-8" ?>
 <ContentPage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
              xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             xmlns:Modus="clr-namespace:Trimble.Modus.Components.Controls.TMInput;assembly=Trimble.Modus.Components"
              Title="Controls Page"
-             x:Class="DemoApp.MainPage">
+             x:Class="DemoApp.MainPage"
+             xmlns:constants="clr-namespace:DemoApp.Constant">
+  <!--<VerticalStackLayout Spacing="10">
```
Comment (rthanga1) [inline DemoApp/DemoApp/MainPage.xaml:8]: Please remove this if not needed

File: DemoApp/DemoApp/Views/MultiLineInputPage.xaml [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+
+        <Modus:MultiLineInput  x:Name="modusInputWithTitle"
+                           TitleText="MultiLine Input"
+                           AutoSize="{Binding IsAutoSize}"
+                           IsEnabled="{Binding IsEnabled}"
+                           IsReadOnly="{Binding IsReadOnly}"
+                           Placeholder="Text Input" />
+
+        <Modus:MultiLineInput TitleText="Success/Error MultiLine Input"
+                          HelperText="Enter more or equal to 100 characters"
```
Comment (rthanga1) [inline DemoApp/DemoApp/Views/MultiLineInputPage.xaml:55]: Please set the valiation to set 10 characters. 100 is too large to verify

File: DemoApp/DemoApp/DemoApp.csproj [hunk 4]
```diff
@@ -82,6 +82,9 @@
 	    <SubType>Code</SubType>
```
Comment (rthanga1) [inline DemoApp/DemoApp/DemoApp.csproj:82]: There is a UI issue in iOS. 
Simulator Screen Shot - iPhone 14 - 2023-06-22 at 15 50 35

**Review by BAL4-R — CHANGES_REQUESTED** [review]
The icon appears too big in windows, can you check this?
image

File: DemoApp/DemoApp/Platforms/Android/AndroidManifest.xml [hunk 5]
```diff
@@ -1,7 +1,7 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
-<manifest xmlns:android="http://schemas.android.com/apk/res/android" package="com.trimble.ModusMAUI">
-  <application android:allowBackup="true" android:icon="@mipmap/appicon" android:roundIcon="@mipmap/appicon_round" android:supportsRtl="true" android:label="Modus UI Control"></application>
-  <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
-  <uses-permission android:name="android.permission.INTERNET" />
-  <uses-sdk android:minSdkVersion="21" android:targetSdkVersion="30" />
-</manifest>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android" package="com.trimble.ModusMAUI" android:versionCode="2" android:versionName="0.0.2">
```
Comment (Sofiya-kumar) [inline DemoApp/DemoApp/Platforms/Android/AndroidManifest.xml:2]: Are these changes for release build?

**Review by Sofiya-kumar — APPROVED** [review]

File: DemoApp/DemoApp/Platforms/Android/AndroidManifest.xml (hunk 5 above)
Comment (ElishaSamPeterPrabhu) [reply DemoApp/DemoApp/Platforms/Android/AndroidManifest.xml:2]: Yes

**Review by BAL4-R — APPROVED** [review]

File: DemoApp/DemoApp/DemoApp.csproj (hunk 1 above)
Comment (ElishaSamPeterPrabhu) [reply DemoApp/DemoApp/DemoApp.csproj]: No, Its removed

File: DemoApp/DemoApp/MainPage.xaml (hunk 2 above)
Comment (ElishaSamPeterPrabhu) [reply DemoApp/DemoApp/MainPage.xaml]: Removed

File: DemoApp/DemoApp/Views/MultiLineInputPage.xaml (hunk 3 above)
Comment (ElishaSamPeterPrabhu) [reply DemoApp/DemoApp/Views/MultiLineInputPage.xaml]: Changed the validation threshold to 10

File: DemoApp/DemoApp/DemoApp.csproj (hunk 4 above)
Comment (rthanga1) [reply DemoApp/DemoApp/DemoApp.csproj:82]: @ElishaSamPeterPrabhu Is this fixed?

File: Trimble.Modus.Components/Controls/TMInput/MultiLineInput.xaml.cs [hunk 6]
```diff
@@ -0,0 +1,66 @@
+namespace Trimble.Modus.Components;
+using Trimble.Modus.Components.Controls.BaseInput;
+
+public partial class MultiLineInput : BaseInput
+{
+    #region Bindable Properties
```
Comment (rthanga1) [inline Trimble.Modus.Components/Controls/TMInput/MultiLineInput.xaml.cs:6]: When AutoSize is disabled, when I keep typing the current cursor position should be disabled.
simulator_screenshot_8D02846E-8555-40FD-901D-C39EE5B96C15

Comment (rthanga1) [reply Trimble.Modus.Components/Controls/TMInput/MultiLineInput.xaml.cs:6]: Looks like this is a known issue in Editor.

**Review by BAL4-R — APPROVED** [review]

File: DemoApp/DemoApp/DemoApp.csproj (hunk 4 above)
Comment (ElishaSamPeterPrabhu) [reply DemoApp/DemoApp/DemoApp.csproj:82]: Yes

**Pull request merged**